packages/ui/src/Fx/Fx.mdx

```diff
@@ -0,0 +1,56 @@
+---
+menu: UI / Hooks
+name: useFx
+---
+
+import { useState } from 'react';
+import { Link, Playground, Props } from 'docz';
+import useFx from './useFx';
+import Popover from './Popover';
+
+# useFx
+
+`useFx` allows you to apply various effects
+
+## Use
+
+```
+import { useFx } from '@rkta/ui';
+...
+
+const MyFunctionComponent = () => {
+  const { phone } = useMedia();
+  const [popoverProps, addEffect] = useFx(['popUp']);
+  return <Paper {...popoverProps} onClose={() => addEffect('fadeDown')} />
```

Comment:
this is nonesense, give real example please

Reply:
updated